### PR TITLE
Bootstrap new envs

### PR DESCRIPTION
**Pull request by simoncross747**
Details of manual process for creating new apps added to README. Also included a scripted version for creating the basic files and structure required.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -88,3 +88,96 @@ Once you have this data, set them as environment variables before running any te
...
+
+2\. Create sub directories for each required environment using the following names:
+
+  **devtest environment**
+  - dev
+  - stage
+  - mock
+
+  **production environment**
+  - testing
```
Comment (glenjamin) [inline README.md]: Can we leave this one out of the docs please? We shouldn't use this in future for any new envs

File: tools/config/main.tf [hunk 2]
```diff
@@ -0,0 +1,29 @@
+variable "app-name" {
+    type = "string"
+    default = "APPNAME"
+}
+
+variable "tags" {
+    type = "map"
+    default {
+        Environment = "Dev"
```
Comment (glenjamin) [inline tools/config/main.tf]: This tag value should be part of the template

File: tools/config/main.tf [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+
+resource "azurerm_resource_group" "group" {
+    name = "${var.app-name}"
+    location = "ukwest"
+    tags = "${var.tags}"
+}
+
+resource "azurerm_storage_account" "storage" {
+    name = "${replace(var.app-name, "-", "")}storage"
```
Comment (glenjamin) [inline tools/config/main.tf]: Could maybe make this `{APPNAME}{ENVNAME}storage` so it lines up with the diginit logic?

File: README.md [hunk 4]
```diff
@@ -88,3 +88,96 @@ Once you have this data, set them as environment variables before running any te
...
+- Azure subscription id (devtest or prod)
+- Azure RM version
+
+3. Copies the azure-provider-config.json to the local directory. This file can then be edited to override defaults.
+
+4. Checks the user is logged in to Azure
+
+5. Sets the Azure subscription based on the value in config.tf.json.
+
+6. Creates the required Azure resource group, storage account and key vault.
```
Comment (glenjamin) [inline README.md]: If diginit makes a key vault, the default `main.tf` should probably also do so.

File: README.md (hunk 1 above)
Comment (simoncross747) [reply README.md]: Removed.

File: tools/config/main.tf (hunk 2 above)
Comment (simoncross747) [reply tools/config/main.tf]: Added.

File: tools/config/main.tf (hunk 3 above)
Comment (glenjamin) [reply tools/config/main.tf]: You're generating the file, so the file generation script could feed in the value - that way if you need to change the app name above, it won't mess with this value.

Comment (simoncross747) [reply tools/config/main.tf]: Added to bootstrap.py.

File: README.md (hunk 4 above)
Comment (simoncross747) [reply README.md]: Added creation of key vault.

**Comment by glenjamin** [comment]
LGTM

**Pull request merged**